From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
Date: Tue, 14 Jun 2022 08:58:21 +0200
Subject: [PATCH] imx8mp_irma6r2_defconfig: Configure writeable list support

Configure the env writable list supporti: CONFIG_ENV_WRITEABLE_LIST. To read
from two sources (nowhere and mmc) the configuration CONFIG_ENV_APPEND must be
enabled. CONFIG_ENV_ACCESS_IGNORE_FORCE prevents force writing. Also activate
the u-boot version variable: CONFIG_VERSION_VARIABLE.

Signed-off-by: Michael Glembotzki <Michael.Glembotzki@iris-sensing.com>
---
 configs/imx8mp_irma6r2_defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configs/imx8mp_irma6r2_defconfig b/configs/imx8mp_irma6r2_defconfig
index fc52e601cd..6aa47245f1 100644
--- a/configs/imx8mp_irma6r2_defconfig
+++ b/configs/imx8mp_irma6r2_defconfig
@@ -32,6 +32,7 @@ CONFIG_OF_BOARD_SETUP=y
 CONFIG_OF_SYSTEM_SETUP=y
 CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=arch/arm/mach-imx/imx8m/imximage-8mp-ddr4.cfg"
 CONFIG_DEFAULT_FDT_FILE="imx8mp-irma6r2.dtb"
+CONFIG_VERSION_VARIABLE=y
 CONFIG_BOARD_LATE_INIT=y
 CONFIG_ARCH_MISC_INIT=y
 CONFIG_BOARD_EARLY_INIT_F=y
@@ -74,6 +75,9 @@ CONFIG_ENV_IS_IN_MMC=y
 CONFIG_SYS_REDUNDAND_ENVIRONMENT=y
 CONFIG_SYS_RELOC_GD_ENV_ADDR=y
 CONFIG_ENV_VARS_UBOOT_RUNTIME_CONFIG=y
+CONFIG_ENV_APPEND=y
+CONFIG_ENV_WRITEABLE_LIST=y
+CONFIG_ENV_ACCESS_IGNORE_FORCE=y
 CONFIG_NET_RANDOM_ETHADDR=y
 CONFIG_REGMAP=y
 CONFIG_SYSCON=y
-- 
2.36.1

